api/swagger/swagger-v1.yaml

```diff
@@ -4708,6 +4906,239 @@ components:
           type: string
         owner_id:
           type: string
+    coin:
+      type: object
+      properties:
```

Comment:
Do you want to mark any of these as required? Otherwise I think they all come back optional in the generated code and that's kind of a pain to work with.